### DISCUSSION TITLE
Bug /wd4141

### DISCUSSION THREAD

**Discussion by davidelacroce89** [kfrlib/kfr#224] [General]
When building on QT6 Creator using g++ I have found an error due to the flag /wd4141 that is specific for VS compilers.
I had to change the line 357 in CMakeList.txt from if (WIN32) to if (MSVC)
With this change I am able to build :)

**Comment by dancasarin** [comment]
Hi,
The compilers supported officially on Windows are MSVC and Clang in MSVC mode (clang-cl). MinGW aren't supported since KFR6. If you feel that this fix help other people with same configuration, pull requests are always welcome.
